lib/index.ts

```diff
@@ -0,0 +1,132 @@
+var QRCode = require('qrcode')
+
+
+export class VendorMetadata {
```

Comment:
Should I use interfaces to define these data-types or is there any specific way to do the same through functions?

Reply:
@HimandriSharma in case of this objects, yes you can simply define object interfaces
The only question is what about the method "toConcise", this can be part of the payment link generation method

@TomasGonzalez how do you suggest to do it?

Reply:
Okay I will look into this!